index.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module 'react-native-toast-message' {
+  import React from 'react';
+  const Toast: {
+    show(options: {
+      type?: 'success' | 'error' | 'info';
+      position?: 'top' | 'bottom';
+      text1: string;
+      text2: string;
+      visibilityTime?: number;
+      autoHide?: boolean;
+      topOffset?: number;
+      bottomOffset?: number;
+      props: { onPress: () => any; guid: string };
+      onShow?: () => {};
+      onHide?: () => {};
+    });
+    hide: (options: { onHide: () => any }) => void;
```

Comment:
Shouldn't options be optional? I had a case now that I only use hide without options.

```suggestion
    hide: (options?: { onHide: () => any }) => void;
```